wp-content/plugins/core/src/Blocks/Blocks_Definer.php

```diff
@@ -114,7 +113,7 @@ public function define(): array {
 				} ),
 			] ),
 
-			Block_Deny_List::class => DI\create()->constructor( DI\get( self::DENY_LIST ) ),
+			Block_Deny_List::class => DI\autowire()->constructor( DI\get( self::DENY_LIST ) ),
```

Comment:
Say I needed to modify `Block_Deny_List`, and I need to inject an instance of something else and it was a concrete object, because it's set to autowire now, I don't have to change anything in the definer at all.